index.html

```diff
@@ -15,22 +15,22 @@
                 </nav>
             </header>
```

Comment:
- [ ] I would kindly suggest this part of the page has the container as an `header` and should not be inside the `section` tag
   - The container of the section is a `header` tag.
        - Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

- [ ] Also I would advice you make the logo on the left hand side be an anchor tag and be wrap inside of `a` tag. And should be inform of a text and not an image
     - Because The logo (if the design has one) is on the left and has a link (a tag).
          - This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- [ ] Lastly the hamburger button should be wrap inside of a nav tag and should be on the right hand side.
   - The hamburger button is on the right, and has a `nav` tag.
      - The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

You can simply restructure your line of code to something like this :point_down: 

```HTML
<header class="header">
   <a href="#">My Logo</a>
   <nav class="navarea">
        <img class="navbar" src="Union.svg">
   </nav>
</header>
```